.github/workflows/data-pipeline-ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v2
         with:
-          python-version: 3.7
+          python-version: 3.9
```

Comment:
The normal way to review a huge PR like this would be commit by commit. However, the commit history here is all over the map, so that's not possible. I don't think we have time for you to rebase it into a more logical order, so I'm going to do what I can from the files view.

Reply:
You mean you would want commits re-ordered like from front-end to back end?

Reply:
Not exactly...let's talk a little more about that offline when we have more time.